_includes/flipflop2.html

```diff
@@ -273,20 +307,29 @@ <h4>Press <strong>f</strong> to begin.</h4>
 						var keys = Object.keys(flipflops);
 
 						for (var i = 0; i < keys.length; i++) {
+							if (keys[i] !== "Z"){
 							summary += '<h3>' + keys[i] + ' = ' + flipflops[keys[i]] + '</h3> ';
+							}
+							else {	
+									summary += '<h3>' + keys[i] + ' = ' +  flipflops[keys[i]] + '</h3> ';
+							}
 						}
 
 						summary += '</p>';
 
 						$('#summary #summa').append(summary);
 						$('#step2').hide();
 						$('#summary').fadeIn();
+						next++;
 					}
+					clearInterval();
+					}, 0 );
```

Comment:
Why did you use setTimeout while giving it a delay of 0 seconds!
Please remove it